docs/standards/rpc-api.md

```diff
@@ -10,14 +10,109 @@ sidebar_position: 10
 This page is under active development.
 
 :::
+The [LUKSO Extension](../guides/browser-extension/install-browser-extension.md) uses the RPC API methods from the table below. The methods are grouped by category: signing methods, standard methods, and LUKSO-specific methods.
 
-The [LUKSO Extension](../guides/browser-extension/install-browser-extension.md) uses new RPC API which are described here.
+## Supported Methods
 
-## Methods
+<table>
+  <tr>
+    <td>Name</td>
+    <td>Type</td>
+  </tr>
+  <tr>
+    <td><a href="#eth_sign">eth_sign</a></td>
+    <td>signing</td>
+  </tr>
+  <tr>
+    <td><a href="#personal_sign">personal_sign</a></td>
+    <td>signing</td>
+  </tr>
+  <tr>
+    <td><a href="#eth_getAccounts">eth_getAccounts</a></td>
+    <td>standard</td>
+  </tr>
+  <tr>
+    <td><a href="#eth_requestAccounts">eth_requestAccounts</a></td>
+    <td>standard</td>
+  </tr>
+  <tr>
+    <td><a href="#eth_sendTransaction">eth_sendTransaction</a></td>
+    <td>standard</td>
+  </tr>
+  <tr>
+    <td><a href="#wallet_switchEthereumChain">wallet_switchEthereumChain</a></td>
+    <td>standard</td>
+  </tr>
+   <tr>
+    <td><a href="#up_addTransactionRelayer">up_addTransactionRelayer</a></td>
+    <td>LUKSO specific</td>
+  </tr>
+   <tr>
+    <td><a href="#up_import">up_import</a></td>
+    <td>LUKSO specific</td>
+  </tr>
+   <tr>
+    <td><a href="#up_generateLsp23Address">up_generateLsp23Address</a></td>
```

Comment:
This method is missing in the description view below.

Reply:
yes, that one is not yet documented. I got some info from Lukas, but it still needs more work. I wanted to add what I have so far and build incrementally

Reply:
Okay cool. Makes sense.

The rest looks great!